src/Handlers/AbstractPaymentMethodHandler.php

```diff
@@ -309,7 +312,6 @@ public function finalize(
         SalesChannelContext $salesChannelContext
     ): void {
         $transactionId = $transaction->getOrderTransaction()->getId();
-        $this->checkoutService->startClient($salesChannelContext->getSalesChannel()->getId());
```

Comment:
this line was unnecessary, the checkout service is instead used in src/Service/PaymentDetailsService.php which is called from `resultHandler->processResult()`

Reply:
I remember this was added as a fix for multiple sales channels configurations, could you confirm if indeed is not needed anymore?

Reply:
we are still taking the sales channel ID at runtime to create the client, the only change I made is to instantiate Checkout with that client instead of calling startClient